docs/ModuleWidgets.md

```diff
@@ -0,0 +1,39 @@
+## Module Widgets
+
+Since LORIS 23.0, LORIS incorporates a concept of "widgets" which
```

Comment:
```suggestion
LORIS incorporates a concept of "widgets" which
```

I don't think it's important to include when the feature was added.

Reply:
I think it's important information